pkg/apis/scope/v0alpha1/types.go

```diff
@@ -35,22 +47,22 @@ type ScopeList struct {
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-type ScopeDashboard struct {
+type ScopeDashboardBinding struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec ScopeDashboardSpec `json:"spec,omitempty"`
+	Spec ScopeDashboardBindingSpec `json:"spec,omitempty"`
 }
 
-type ScopeDashboardSpec struct {
+type ScopeDashboardBindingSpec struct {
 	DashboardUIDs []string `json:"dashboardUids"`
 	ScopeUID      string   `json:"scopeUid"`
```

Comment:
```suggestion
	Scope      string   `json:"scope"`
	Dashboards []string `json:"dashboards"`
```
In k8s land these will actually be the "names" so lets keep UID out of it to avoid confusion

Reply:
@bfmatei I think you caught this in your PR. but FYI

Reply:
Yup! Already implemented in my PR